GPX/GPXTrailsTrackPointExtensions.m

```diff
@@ -0,0 +1,74 @@
+//
+//  GPXTrailsTrackPointExtensions.m
+//  GPX
+//
+//  Created by Jan Weitz on 27.04.2015
+//
+//
+
+#import "GPXTrailsTrackPointExtensions.h"
+#import "GPXElementSubclass.h"
+
+NSString *const kElementHorizontalAcc = @"trailsio:hacc";
+NSString *const kElementVerticalAcc = @"trailsio:vacc";
+
+@interface GPXTrailsTrackPointExtensions ()
+
+@property (nonatomic) NSString *horizontalAccuracyString;
+@property (nonatomic) NSString *verticalAccuracyString;
+
+@end
+
+@implementation GPXTrailsTrackPointExtensions
+
+- (id)initWithXMLElement:(GPXXMLElement *)element parent:(GPXElement *)parent {
+    self = [super initWithXMLElement:element parent:parent];
+    
+    if (self) {
+        _horizontalAccuracyString = [self textForSingleChildElementNamed:kElementHorizontalAcc xmlElement:element];
+        _verticalAccuracyString = [self textForSingleChildElementNamed:kElementVerticalAcc xmlElement:element];
+    }
+    
+    return self;
+}
+
+#pragma mark - Public methods
+- (void)setHorizontalAccuracy:(NSNumber *)horizontalAccuracy {
+    _horizontalAccuracyString = [NSString stringWithFormat:@"%.2f", [horizontalAccuracy doubleValue]];
+}
+
+- (void)setVerticalAccuracy:(NSNumber *)verticalAccuracy {
+    _verticalAccuracyString = [NSString stringWithFormat:@"%.2f", [verticalAccuracy doubleValue]];
+}
+
+- (NSNumber *)horizontalAccuracy {
+    if (!_horizontalAccuracyString.length) {
+        return nil;
+    }
+    
+    return [NSNumber numberWithFloat:[GPXType decimal:_horizontalAccuracyString]];
+}
+
+- (NSNumber *)verticalAccuracy {
+    if (!_verticalAccuracyString.length) {
+        return nil;
+    }
+    
+    return [NSNumber numberWithFloat:[GPXType decimal:_verticalAccuracyString]];
```

Comment:
vllt. in einem anderen PR, wenn wir hiermit fertig sind. Halten wir das changeset mal klein für den Moment.